quizzes/midterm.md

```diff
@@ -64,52 +89,90 @@
 	    giant_ants % 2 # You small little guys have trouble taking anyone down.
 	  end
 	end
-	```
+```
 
+```
+125
+```
+10: Demonstrate using multiple logical operators (||, &&)
 
-1. Demonstrate using multiple logical operators (||, &&)
+```lang-ruby
+	variable = true
+	if variable == || variable == false && variable == True
+		puts "always runs true becuase if the first part is true it does not go to the second part."
+	end 
+```
 
 
-1. What is the result of calling sort?
+11: What is the result of calling sort?
 
-	```ruby
+```lang-ruby
 	["one fish", "two fish", "red fish", "blue fish"].sort
-	```
+```
+
+```lang-ruby
+**["blue fish" , "one fish" , "two fish"]**
+```
 
 
-1. Write a hash that has the following keys :name, :super_power, :weakness
+12: Write a hash that has the following keys :name, :super_power, :weakness
 
+```lang-ruby
+hash = {:keys , :super_power, :weakness}
+```
 
-1. T/F You can set a default value for a Hash
+13: **T**/F You can set a default value for a Hash
 
 
 
-1. What is the resulting hash?
+14: What is the resulting hash?
 
-	```ruby
+```lang-ruby
 	my_hash = { "a" => 100, "b" => 200, "c" => 300 }
 	my_hash.select {|key,value| key < "c"}
-	```
+```
 
-1. Write code that iterates through the numbers array and out`puts` the `KEY` for `each` of the values in the array.
+```lang-ruby
+	it selects "a" and "b"
+```
 
-	```ruby
+15: Write code that iterates through the numbers array and out`puts` the `KEY` for `each` of the values in the array.
+
+```lang-ruby
 	numbers = ['one', 'two', 'three']
-	```
+```
+
+```lang-ruby
+	puts numbers.map
+```
 
-1. Write code that `maps` the fruits array to a new array where all the values are `reverse`d. Eg. "apples" = "selppa"
+16: Write code that `maps` the fruits array to a new array where all the values are `reverse`d. Eg. "apples" = "selppa"
```

Comment:
10/10 points. I'll give you full credit because you got the general idea. I was hinting at using the `map` method.